test/run-k8s-integration.sh

```diff
@@ -12,7 +12,7 @@ if [ "$K8S_E2E_SCRIPT_PARENT_DIR" != "$PKGDIR" ]; then
   exit 1
 fi
 
-readonly overlay_name="${GCE_FS_OVERLAY_NAME:-stable}"
+readonly overlay_name="${GCE_FS_OVERLAY_NAME:-stable-master}"
```

Comment:
with this change, [pull-gcp-filestore-csi-driver-kubernetes-integration](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/gcp-filestore-csi-driver/gcp-filestore-csi-driver-config.yaml#L71) will start running test against stable-master